frontend/__tests__/dummy.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { render } from 'enzyme';
+
+import LoginLogoutButton from '../components/LoginLogoutButton';
+
+describe("test the login/logout button", () =>{
+    it("renders the button", async () => {
+        try {
+            render(<LoginLogoutButton/>);
```

Comment:
shouldn't this be `expect(stuff).to.have.lengthOf(1);` ?

Reply:
In this case no, as the purpose of the test is not to test the component specifically but to test the usage of the Enzyme library (here, render(...)).

But in a 'proper' frontend test, you might be correct. I'll have to look into it.